repo/packages/K/kafka/0/marathon.json

```diff
@@ -0,0 +1,13 @@
+{
+  "id": "kafka",
+  "cpus": 0.5,
+  "mem": 256,
+  "instances": 1,
+  "cmd": "java -jar kafka-mesos*.jar scheduler --master=zk://master.mesos:2181/mesos --zk=master.mesos:2181 --api=http://localhost:$PORT0",
```

Comment:
Nope. It will bind to 0.0.0.0. 
But the URL, that Executor will use to download deps would be incorrect. Need to fix.